web-admin/tests/setup/setup.ts

```diff
@@ -0,0 +1,200 @@
+import { chromium, expect } from "@playwright/test";
+import axios from "axios";
+import { spawn } from "child_process";
+import dotenv from "dotenv";
+import path from "path";
+import { fileURLToPath } from "url";
+import { writeFileEnsuringDir } from "../utils/fs";
+import { execAsync, spawnAndMatch } from "../utils/spawn";
+import type { StorageState } from "../utils/storage-state";
+import { test as setup } from "./base";
+import {
+  ADMIN_STORAGE_STATE,
+  RILL_DEVTOOL_BACKGROUND_PROCESS_PID_FILE,
+} from "./constants";
+import { cliLogin } from "./fixtures/cli";
+
+setup(
+  "should start services, log-in a user, and deploy a project",
+  async () => {
+    const timeout = 240_000;
```

Comment:
I see there was a commit to bump this. With more and more things added during setup this would need bumping now and then. Any way to use a very high timeout with escape hatch for error logged?

Reply:
So, the runtime's ingest of the test project data is what is at risk of timing out. Rather than increasing the timeout, I'd actually like to use a test project with smaller data. 

While, in the long-run, we'll probably want to test against multiple projects, in the short-run, I'd like to create one project that can serve many test cases. I haven't yet built that project, and I don't think any of the example projects really work at the moment. But because this PR has been open for so long, I'd like to do it in a follow-up.